Event_Display/ClusterShapeDisplay.cpp

```diff
@@ -0,0 +1,44 @@
+#include "TCanvas.h"
+#include "TH2D.h"
+#include "TStyle.h"
+#include <bitset>
+#include <iostream>
+
+void fillBinaryMatrix(TH2D *hist, const uint8_t *ExampleArray, int size,
+                      int max_length, int xClusterPosition,
+                      int yClusterPosition) {
+  for (int i = 0; i < size; ++i) {
+    std::bitset<8> bits(ExampleArray[i]);
+    for (int j = 0; j < max_length; j++) {
+      hist->Fill(xClusterPosition + j, yClusterPosition - i - 1,
+                 bits[max_length - 1 - j]); // Adjusting the y-position
+    }
+  }
+}
+
+void ClusterShapeDisplay() {
+  // need to link t cluster class
+  // uint8_t ExampleArray[] = {16, 48, 48, 224, 32, 0, 0, 0};// Example array
+  uint8_t ExampleArray[] = {192, 192, 80, 248, 16, 16, 16, 16};
+  int size = sizeof(ExampleArray) / sizeof(ExampleArray[0]);
+  int max_length = 8;
+  // int xClusterPosition = 123;
+  // int yClusterPosition = 456;
+  int xClusterPosition = 379;
+  int yClusterPosition = 28;
+
+  TH2D *hist = new TH2D("Cluster", "Cluster", max_length, xClusterPosition,
+                        xClusterPosition + max_length, size,

```

Comment:
```suggestion
                        xClusterPosition + max_length, MAX_CLUSTER_SIZE,
```

Reply:
Can you try if this works also here? It will need my PR to be merged tho

Reply:
I have change other staff, I can't use MAX_CLUSTER_SIZE without your PR maybe.